index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/e520ead38a.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="header">
+        <nav class="navbar">
+            <div class="web-name">Sohail</div>
+            <a href="#" class="bar-button">
+                <i class="fa-solid fa-bars"></i>
+            </a>
+        </nav>
+    </div>
+    <main>
+        <div class="details">
+            <h1>I'm Sohail Glad to see you!</h1>
+            <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <h3>LET'S CONNECT</h3>
+            <div class="social-links">
+                <a href="#" class="social-icons"><i class="fa-brands fa-twitter"></i></a>
+                <a href="#" class="social-icons"><i class="fa-brands fa-linkedin"></i></a>
+                <a href="#" class="social-icons"><i class="fa-brands fa-facebook"></i></i></a>
+                <a href="#" class="social-icons"><i class="fa-brands fa-square-github"></i></a>
+            </div>
```

Comment:
- [ ] I think it would be better to use an order list of `ul` and `li` instead of `div` for a semantic reason.

Example
```
<ul>
    <li>
        <a href="">
            <i></i>
        </a>
    </li>
    <li>
        <a href="">
            <i></i>
        </a>
    </li>
   .
   .
   .
   
</ul>
```